crates/type-system/.gitignore

```diff
@@ -0,0 +1,6 @@
+/target
+**/*.rs.bk
+Cargo.lock
+bin/
+pkg/
+wasm-pack.log
```

Comment:
Can we hoist this to top-level `.gitignore`?

Reply:
That's been done https://github.com/blockprotocol/blockprotocol/pull/503/commits/9d0cf95cc83e95a1c2ed20e4540796c417207b3b

Let me know if that's not how you intended it to be